include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3329,6 +3341,86 @@ struct OrtApi {
   ORT_API2_STATUS(AddExternalInitializers, _In_ OrtSessionOptions* options,
                   _In_reads_(input_len) const char* const* initializer_names,
                   _In_reads_(input_len) const OrtValue* const* initializers, size_t initializers_num);
+
+  /** \brief: Create attribute of onnxruntime operator
+  * 
+  * \param[in] name of the attribute
+  * \param[in] data of the attribute
+  * \param[in] data length
+  * \param[in] data type
+  * \param[out] attribute that has been created, which must be released by OrtApi::ReleaseOpAttr
+  * 
+  * \since Version 1.12.
+  */
+  ORT_API2_STATUS(CreateOpAttr,
+                  _In_ const char* name,
+                  _In_ const void* data,
+                  _In_ int len,
+                  _In_ OrtOpAttrType type,
+                  _Outptr_ OrtOpAttr** op_attr);
+
+  /* \brief: Release op attribute
+  *
+  * \param[in] attribute created by OrtApi::CreateOpAttr
+  * 
+  * \since Version 1.12.
+  */
+  ORT_CLASS_RELEASE(OpAttr);
+
+  /** \brief: Create onnxruntime native operator
+  * 
+  * \param[in] kernel info 
+  * \param[in] operator name
+  * \param[in] operator domain
+  * \param[in] operator opset
+  * \param[in] name of the type contraints, such as "T" or "T1"
+  * \param[in] type of each contraints
+  * \param[in] number of contraints
+  * \param[in] attributes used to initialize the operator
+  * \param[in] number of the attributes
+  * \param[out] operator that has been created
+  * 
+  * \since Version 1.12.
+  */
+  ORT_API2_STATUS(CreateOp,
+                  _In_ const OrtKernelInfo* info,
+                  _In_ const char* op_name,
+                  _In_ const char* domain,
+                  _In_ int version,
```

Comment:
nit: can name this to opset_version

Reply:
> nit: can name this to opset_version

Sorry, missing the comment, will rename it later.